setup/src/Magento/Setup/Console/Command/AdminUserCreateCommand.php

```diff
@@ -123,4 +124,31 @@ public function validate(InputInterface $input)
 
         return $errors;
     }
+
+    /**
+     * Get admin user data
+     *
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @return void
+     */
+    protected function setMissingValues(InputInterface &$input, OutputInterface $output)
+    {
+        if (!method_exists($input, 'promptForOption')) {
```

Comment:
sorry for not being more specific earlier, but better way to achieve this is to introduce new interface, i.e. InputInteractiveInterface, which extends InputInterface. Then here either check is $input an instance of this interface or even better do type hinting to InputInteractiveInterface in the method signature. The latter is not always possible, so either way.